tartare/interfaces/coverage_data_source_subscription.py

```
@@ -65,3 +66,14 @@ def post(self, coverage_id):
             return {'message': str(e)}, 400
 
         return {'coverages': schema.CoverageSchema().dump([coverage], many=True).data}, 200
+
+    def delete(self, coverage_id, data_source_id):
+        coverage = models.Coverage.get(coverage_id)
+        if coverage is None:
+            return {'message': 'Bad coverage {}.'.format(coverage_id)}, 404
+
+        if data_source_id not in coverage.data_sources:
+            return {'message': 'Unknown data_source_id attribute in uri.'}, 404
+
+        coverage.remove_data_source(coverage_id, data_source_id)
```

i can let it raise so we catch it top level and return a 500 response no?